src/gpu/intel/jit/codegen/codegen_extensions.hpp

```diff
@@ -13,24 +13,35 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *******************************************************************************/
+#ifndef GPU_INTEL_JIT_CODEGEN_CODEGEN_EXTENSIONS_HPP
```

Comment:
Can we drop the prefix in the file name? or is it because of some name clashing?
`codegen/codegen_extensions.hpp` -> `codegen/extensions.hpp`